west.yml

```diff
@@ -261,7 +261,7 @@ manifest:
     - name: hostap
       repo-path: hostap
       path: modules/lib/hostap
-      revision: 83574f533fb5c6808af0d9872741d72d48be2f98
+      revision: pull/12/head
```

Comment:
This should be the `oldest` commit as without this other commits won't build.

Reply:
not quite understand this comment.
https://github.com/zephyrproject-rtos/hostap/pull/12 is new PR on top of latest commit, I think.

Reply:
the ordering would be (old to new):

* Bring in changes from hostap as these are needed before we use them e.g., a new API added
* Do the corresponding changes in `modules/hostap` e.g., call new API

this way build isn't broken if someone checks out either commit. But if the changes have circular dependency, this cannot be fixed and order doesn't matter then.